app/assets/stylesheets/standards/_classes.sass

```diff
@@ -12,3 +12,28 @@
 
 .bg-pink
   background-color: $yellow-color
+
+.bg-light-gray
+  background-color: $light-gray
+
+.bg-medium-gray
+  background-color: $medium-gray
+
+.bg-dark-gray
+  background-color: $dark-gray
+
+.mb-1rem
+  margin-bottom: 1rem
+
+.ml-onehalf
```

Comment:
You have to decide to a naming pattern and stick to it: `mb-1rem` and `ml-onehalf` have different naming pattern. Also `onehalf` means `0.5`. I know you didn't want to make the name long, but accurate long names are better than inaccurate short ones :)